nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -246,12 +246,12 @@ def __init__(self, cfg: DictConfig, trainer: Trainer, no_lm_init=True):
         self.use_fsdp = cfg.get('fsdp', False)
 
     def setup_transformer_engine_tp_groups(self):
-        """ This should be called after model parallel groups have been initialized
```

Comment:
please go though the changes and undo anything that's not necessary

Reply:
from offline discussion, keep it for now since it is automatically applied after pushing the code https://github.com/NVIDIA/NeMo/pull/9169/commits/3cc0432ccb9dcab9b1d9bb52d17ad912fc21fbdf